ding2.make

```diff
@@ -428,6 +428,9 @@ projects[ask_vopros][download][tag] = "1.5"
 projects[xautoload][subdir] = "contrib"
 projects[xautoload][version] = "5.7"
 
+projects[lazy_pane][subdir] = "contrib"
+projects[lazy_pane][version] = "7.x-1.2"
```

Comment:
@martinb14: Note that the build on CircleCI fails because the version definition is not correct. Is should be "1.2".

![ding2 1628 - circleci 2016-12-15 10-18-31](https://cloud.githubusercontent.com/assets/73966/21218178/e52f7cc8-c2af-11e6-8053-9b21d45452b9.png)